java-agent/instrumentation/reactor-core-3.1/src/main/java8/io/opentelemetry/auto/instrumentation/reactor/core/FluxAndMonoSubscribeAdvice.java

```diff
@@ -1,12 +1,12 @@
 package io.opentelemetry.auto.instrumentation.reactor.core;
 
-import static io.opentelemetry.auto.instrumentation.api.AgentTracer.activateSpan;
-
-import io.opentelemetry.auto.instrumentation.api.AgentScope;
-import io.opentelemetry.auto.instrumentation.api.AgentSpan;
+import io.opentelemetry.auto.instrumentation.api.SpanScopePair;
+import io.opentelemetry.trace.Span;
 import net.bytebuddy.asm.Advice;
 import reactor.core.CoreSubscriber;
 
+import static io.opentelemetry.auto.instrumentation.reactor.core.ReactorCoreDecorator.TRACER;
```

Comment:
Can you check your [Intellij formatting settings](https://github.com/open-telemetry/opentelemetry-auto-instr-java/blob/master/CONTRIBUTING.md#intellij-idea)?